packages/pretty-format/src/__tests__/ImmutablePlugins-test.js

```diff
@@ -0,0 +1,510 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+ /* eslint-disable max-len */
+
+'use strict';
+
+const React = require('react');
+const prettyFormat = require('../');
+const Immutable = require('immutable');
+const ReactElementPlugin = require('../plugins/ReactElement');
+const ReactTestComponentPlugin = require('../plugins/ReactTestComponent');
+const ImmutablePlugins = require('../plugins/ImmutablePlugins');
+
+function assertImmutableObject(actual, expected, opts) {
+  expect(
```

Comment:
Could we use `expect.extend` instead?

See http://facebook.github.io/jest/docs/expect.html#expectextendmatchers

```
expect(immutableValue).toPrettyPrintTo('Immutable.OrderedSet')
```

or something like this?